ngshare/ngshare.py

```diff
@@ -699,7 +790,7 @@ def get(self):
 class MyApplication(Application):
     'Custom application for ngshare'
     def __init__(self, prefix, db_url, storage_path, debug=False,
-                 autoreload=True):
+                 admin=(), autoreload=True):
```

Comment:
Should be a list `[]` rather than a tuple `()`? Probably doesn't matter

Reply:
Pylint complains about it, so maybe I will just use tuple. 
```
$ pylint ngshare.py
************* Module ngshare.ngshare
ngshare.py:798:4: W0102: Dangerous default value [] as argument (dangerous-default-value)

-------------------------------------------------------------------
Your code has been rated at 9.98/10 (previous run: 10.00/10, -0.02)
$ 
```